.github/workflows/ubuntu-build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         with:
           repository: sparcians/map
           path: map
-          ref: map_v1.1.1
+          ref: map_v2.0.0
```

Comment:
Is this ready to merge with the PR in sparta now merged as well? This checks out map_v2.0.0 tag of sparta, but I guess that can stay?

Reply:
I'd like to make a release of Sparta with that change OR we can just use `map_v2` development branch (will cherry pick your merge).

Reply:
Pushed a change to map_v2.  Once regression passes, I'll merge this in!